README.md

```diff
@@ -0,0 +1,303 @@
+# frontend-build
+
+## Warning: Package Moved
+
+While the Paragon design system will continue to receive updates, these updates
+will no longer be published at the @edx/paragon NPM package. Instead, the
```

Comment:
```suggestion
will no longer be published at the @edx/frontend-build NPM package. Instead, the
```

Reply:
ugh, it didn't load your comment, I'll make a follow-up PR to fix this before I merge the update.